source/man/option-A.inc

```diff
@@ -0,0 +1,13 @@
+**-A**
+   If the **-A** option is present, the *command* will be performed for
+   all users. Using this option in combination with system users from
+   **userdb { driver = passwd }** is not recommended, because it
+   contains also users with a lower UID than the one configured with the
+   *first_valid_uid* setting.
+
+When the SQL userdb module is used make sure that the *iterate_query*
+setting in *@pkgsysconfdir@/dovecot-sql.conf.ext* matches your database
+layout. When using the LDAP userdb module, make sure that the
+*iterate_attrs* and *iterate_filter* settings in
+*@pkgsysconfdir@/dovecot-ldap.conf.ext* match your LDAP schema.
+Otherwise **doveadm**\ (1) will be unable to iterate over all users.
```

Comment:
In the current manpages this paragraph is indented the same as the first paragraph. In this current form, this is indented with the `**-A**` option.

To fix this I guess you need to align the paragraph with the first paragraph.